### DISCUSSION TITLE
Best practice for rendering max 200K potentials text

### DISCUSSION THREAD

**Discussion by giodnt** [pixijs/pixijs#7940] [Q&A]
Hi,

hi, 
I’m building a gantt chart which can contain over 200k objects.  I have not encountered problems with the objects, but with the text of each of them. How can you proceed in these cases?

Many thanks

**Comment by ivanpopelyshev** [comment]
BitmapText only. if you need scale  - use SDF or MSDF.

**Comment by ivanpopelyshev** [comment]
also, 200k should be culled. really. 

Read my post here: https://github.com/pixijs/pixijs/issues/7934#issuecomment-955658630

i posted it like 50 times already. People dont get it - PixiJS provides **low-level** part of solution. If you ask to render 200k elements withotu a problem - you have to write good **high-level** too! Otherwise , you as a coder can be easily replaced by neural network.

**Reply by giodnt** [reply]
Please show me how, if it's that easy for you…

**Reply by ivanpopelyshev** [reply]
https://www.facebook.com/playthehousehold/ - look at top users, 10k animated objects on their fields.

There's no 200k texts demo for pixi. there's not even 10k texts demo, because people who did that - didnt share results. You can do it and then share it.
